bento_beacon/config_files/config.py

```diff
@@ -114,6 +114,7 @@ class Config:
     KATSU_BEACON_SEARCH = "/api/beacon_search"
     KATSU_SEARCH_OVERVIEW = "/api/search_overview"
     KATSU_PRIVATE_OVERVIEW = "/api/overview"
+    KATSU_PUBLIC_OVERVIEW = "api/public_overview"
```

Comment:
inconsistent slash prefix with other variables

Reply:
huh

Reply:
fixed, looks like urlunsplit was covering for me here